File: ruby/001/main.rb
=begin
    ProjectEuler 001
    Ruby / main.rb
    copyright (c) 2015 Susisu

    Ruby 2.0.0p481
    `ruby main.rb`
=end

def main
    sum = 0
    (1..999).each{|n|
        if n % 3 == 0 || n % 5 == 0 then
            sum += n
        end
    }
    puts sum
end

main()

File: php/001/main.php
<?php
/*
 * ProjectEuler 001
 * PHP / main.php
 * copyright (c) 2015 Susisu
 *
 * PHP 5.4.30
 * `php main.php`
 */

function main() {
    $sum = 0;
    for ($n = 1; $n < 1000; $n++) {
        if ($n % 3 === 0 || $n % 5 === 0) {
            $sum += $n;
        }
    }
    echo "$sum\n";
}

main();
